frontend/wallet/src/render/i18n/translations/vn.json

```diff
@@ -0,0 +1,118 @@
+[
+  {
+    "id": "account-settings",
+    "defaultMessage": "account settings",
+    "message": "cài đặt tài khoản"
```

Comment:
Tbh, I just used google translate. A fluent Vietnamese speaker should come and do some real vetting lol.